councilmatic/settings_jurisdiction.py

```diff
@@ -278,7 +278,8 @@
     'tony-tavares-a81515b1c413': {'source': '', 'image': 'manual-headshots/tony-tavares.jpg'},
     'holly-mitchell-68d838612b17': {'source': '', 'image': 'manual-headshots/holly-mitchell.jpg'},
     'fernando-dutra-19bb1bb05d11': {'source': '', 'image': 'manual-headshots/fernando-dutra.png'},
-    'tim-sandoval-387c30ca1882': {'source': '', 'image': 'manual-headshots/tim-sandoval.jpeg'}
+    'tim-sandoval-387c30ca1882': {'source': '', 'image': 'manual-headshots/tim-sandoval.jpeg'},
+    'stephanie-wiggins': {'source': '', 'image': 'stephanie-wiggins.jpg'},
```

Comment:
A couple of things:

- `stephanie-wiggins` should be her slug from the production site, which includes the string of letters and numbers. You can find the slug by going on the live site to the Committees page, then clicking into one of the committees, then clicking on Stephanie Wiggins' link in the member table.
- Is this the correct filepath and file type? The image you added is `lametro/static/images/manual-headshots/stephanie-wiggins.jpeg`.